Broken compilation from bad import of FormData

**Description**

Hi

I think the problem that was resolved with https://github.com/jdalrymple/gitbeaker/issues/417 is back.
I think it's a regression from the monorepo commit the 1 Feb 2020

` node_modules/@gitbeaker/requester-utils/node_modules/form-data/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag
    
    1 import FormData from 'form-data';
             ~~~~~~~~
    
      node_modules/@gitbeaker/requester-utils/node_modules/form-data/index.d.ts:10:1
        10 export = FormData;
           ~~~~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.

`

**Expected behaviour**
build succes

**Actual behaviour**
build error

**Possible fixes**
In the RequesterUtils.ts replace 
`import FormData from 'form-data';	`
By
`import * as FormData from 'form-data';`

This one isnt that straight forward. Replacing with: 

```
 import * as FormData from 'form-data';
```

Just returns another error:
```
src/RequesterUtils.ts:44:53 - error TS2359: The right-hand side of an 'instanceof' expression must be of type 'any' or of a type assignable to the 'Function' interface type.

44   if (typeof body === 'object' && !(body instanceof FormData)) {
                                                       ~~~~~~~~
```

--- a/packages/gitbeaker-requester-utils/src/RequesterUtils.ts
+++ b/packages/gitbeaker-requester-utils/src/RequesterUtils.ts
@@ -1,7 +1,7 @@
 import { Agent } from 'https';
-import FormData from 'form-data';
 import { decamelizeKeys } from 'xcase';
 import { stringify } from 'query-string';
+import FormDataC from 'form-data';
 
 // Types
 export interface RequesterType {
@@ -41,7 +41,7 @@ export function defaultRequest(
 
   if (sudo) headers.sudo = sudo;
 
-  if (typeof body === 'object' && !(body instanceof FormData)) {
+  if (typeof body === 'object' && !(body instanceof FormDataC)) {
     bod = JSON.stringify(decamelizeKeys(body));
     headers['content-type'] = 'application/json';
   } else {

--- a/packages/gitbeaker-requester-utils/test/unit/RequesterUtils.ts
+++ b/packages/gitbeaker-requester-utils/test/unit/RequesterUtils.ts
@@ -1,5 +1,5 @@
 import FormData from 'form-data';
-import { createInstance, defaultRequest } from '../../src/RequesterUtils';
+import { createInstance, defaultRequest, modifyServices } from '../../src/RequesterUtils';
 
 const methods = ['get', 'put', 'delete', 'stream', 'post'];
 
@@ -42,6 +42,14 @@ describe('defaultRequest', () => {
     });
 
     expect(headers.sudo).toBeUndefined();
+  });
+
+  it('should assign the sudo property if passed', async () => {
+    const { headers } = defaultRequest(service, {
+      sudo: 'testsudo',
+    });
+
+    expect(headers.sudo).toBe('testsudo');
   });
 
   it('should default searchParams to an empty string if undefined', async () => {
@@ -87,5 +95,21 @@ describe('createInstance', () => {
 
       expect(handler).toBeCalledWith(testEndpoint, {});
     });
+  });
+});
+
+describe('modifyServices', () => {
+  it('should modify class with default properties', async () => {
+    class A {
+      constructor({ x, y }: { x?: number; y?: number } = {}) {
+        this.x = x;
+        this.y = y;
+      }
+    }
+
+    const { A: B } = modifyServices({ A }, { x: 3 });
+    const b = new B();
+
+    expect(b.x).toBe(3);
   });
 });

